Apps/release-planner/overview.md

```diff
@@ -0,0 +1,247 @@
+# Release Planner Solution
```

Comment:
Please rename this file to readme.md if this is what should be displayed on the main page.

Reply:
done